### DISCUSSION TITLE
Feature requests

### DISCUSSION THREAD

**Discussion by Roye7777777** [gwleuverink/clickup-time-tracker#16] [Ideas]
- Add support for CMD/CTRL + A; CMD/CTRL + C; CMD/CTRL + X; CMD/CTRL + V in description and title field once the input modal is active
- Ability to descriptions once created
- Unsaved input is emptied on blur at the moment. Is that intentional?
- Mogelijkheid naar item klikbaar te maken en de card in kwestie zelf te openen in de browser

**Comment by gwleuverink** [comment]
Added in next release except for the input clearing thing. Not sure what you mean. Is this still an issue in the latest version (main branch)
